apps/protocol-frontend/src/hooks/useContributionCount.ts

```diff
@@ -0,0 +1,57 @@
+import { useQuery } from '@tanstack/react-query';
+import { useUser } from '../contexts/UserContext';
+import { subWeeks } from 'date-fns';
+
+type ContributionCountArgs = {
+  startDate: Date;
+  endDate: Date;
+  guildIds?: number[] | null;
+  excludeUnassigned?: boolean;
+};
+
+const useContributionCountInRange = (args: ContributionCountArgs) => {
+  const { userData, govrnProtocol: govrn } = useUser();
+
+  const { isLoading, isFetching, isError, error, data } = useQuery(
+    ['ContributionGetCount', args],
```

Comment:
Not a big deal, but the other queries start with a lowercase such as `contributionGetCount`